### PR TITLE
Adding PR Template

### PR DESCRIPTION
**Pull request by CheezItMan**
Added Basic PR Template.

Possibly we should add a question on Post vs Get requests, but Edamam only does get requests. 

Also we could break it into waves, but they'll have the initial knowledge to work the project after the initial API lesson.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
 # Recipe API Consumer
 
+## Purpose
+Create a web application that will display recipes based on a search term. We will utilize an API from Edamam called the [Recipe Search API](https://developer.edamam.com/edamam-recipe-api).
+
+This is an individual, [stage 2](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md) project.
```
Comment (kariabancroft) [inline README.md:6]: 👍

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+# API Muncher
+Congratulations! You're submitting your assignment!
+
+## Comprehension Questions
+| Question | Answer    |
+| :------------- | :------------- |
+| How did you go about exploring the Edamam API, how did you try querying the API? |  |
+| Describe your API Wrapper.  How did you decide on the methods you created? |  |
+| Describe how you approached testing your API Wrapper, what edge cases do you take? |  |
```
Comment (kariabancroft) [inline .github/PULL_REQUEST_TEMPLATE]: Maybe this could be simplified to "Describe an edge case or failure case test you wrote for your API Wrapper"

File: README.md (hunk 1 above)
Comment (kariabancroft) [reply README.md:6]: Should we add a Trello requirement since we are not providing the project in waves to practice these skills?

File: .github/PULL_REQUEST_TEMPLATE [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+| Explain how VCR aids in testing an API. |  |
+| What is the Heroku URL of your deployed application? |  |
```
Comment (kariabancroft) [inline .github/PULL_REQUEST_TEMPLATE]: This isn't in the requirements so if we want them to deploy it we should add it

File: feedback.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+| Controller testing |  |
+| **Styling** |  |
+| Foundation Styling for responsive layout |  |
+| Search View |  |
+| List View |  |
+| Short View |  |
+| **Styling** |  |
+| Foundation is used appropriately |  |
+| Look and feel is similar to the original |  |
+|  **Overall** |  |
```
Comment (kariabancroft) [inline feedback.md]: We should add something for API attribution (since that's in the requirements for using the API)

Comment (kariabancroft) [reply feedback.md]: We should ensure the API key isn't saved in the cassettes (noted from dan's original spreadsheet from C6)

Comment (kariabancroft) [reply feedback.md]: Show view opens recipe in a new tab

**Review by kariabancroft — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (CheezItMan) [reply README.md:6]: Added

File: .github/PULL_REQUEST_TEMPLATE (hunk 3 above)
Comment (CheezItMan) [reply .github/PULL_REQUEST_TEMPLATE]: Added

File: feedback.md (hunk 4 above)
Comment (CheezItMan) [reply feedback.md]: added.

File: .github/PULL_REQUEST_TEMPLATE (hunk 2 above)
Comment (CheezItMan) [reply .github/PULL_REQUEST_TEMPLATE]: added

**Review by kariabancroft — APPROVED** [review]
